core/templates/dev/head/exploration_editor/EditorServices.js

```diff
@@ -673,6 +673,65 @@ oppia.factory('explorationStatesService', [
       editorContextService, validatorsService, newStateTemplateService,
       explorationGadgetsService) {
     var _states = null;
+
+    // Maps backend names to the corresponding frontend dict accessor lists.
+    var PROPERTY_REF_DATA = {
+      answer_groups: ['interaction', 'answer_groups'],
+      confirmed_unclassified_answers: [
+        'interaction', 'confirmed_unclassified_answers'],
+      content: ['content'],
+      default_outcome: ['interaction', 'default_outcome'],
+      param_changes: ['param_changes'],
+      fallbacks: ['interaction', 'fallbacks'],
+      widget_id: ['interaction', 'id'],
```

Comment:
No, this is for backend compatibility. I think the key used in the changelist is still widget_id.